### DISCUSSION TITLE
Feature: Add more information to git log

### DISCUSSION THREAD

**Discussion by marsianer** [Kozea/Radicale#1363] [Ideas]
It would be helpful if there are more information in the git log.

- Summary of the calendar event
- Full name of the contact
- Type of Change (Create, Modify, Delete)

I only know the user variable: %(user)
`hook = git add -A && (git diff --cached --quiet || git commit -m "Changes by "%(user)s)`

```
Contact John Doe [created]
User: marsianer

Contact  John Doe [modified]
User: marsianer

Contact  John Doe [deleted]
User: marsianer

Contact  Jane Doe [modified]
User: marsianer

Event Meeting with Jane Doe - 2023-10-08-01-30 [created]
User: marsianer
```

**Comment by pbiering** [comment]
for your feature request a magic extension has to parse the related iCal or vCard file and also has to recognize whether a new file was created or one was deleted...I would assume this is not that easy. Feel free to develop a dedicated hook script which does all what you want.
